packages/app/src/LogTable.tsx

```diff
@@ -13,6 +13,7 @@ import cx from 'classnames';
 import { Button, Modal } from 'react-bootstrap';
 import stripAnsi from 'strip-ansi';
 import { CSVLink } from 'react-csv';
+import { curry } from 'lodash';
```

Comment:
optional nit: we don't have any tree-shaking for lodash built in the build chain so it'd be preferable to do `import curry from lodash/curry`. However, we already import all of lodash in a few other components and it needs to be made consistent across the entire code base.

We can merge this as-is, since we're already inconsistent, and I made a follow-up issue addressing the broader issue in our code base https://github.com/hyperdxio/hyperdx/issues/36

Reply:
Thanks for taking a look! I just pushed the update
I could work on #36 as well, if you don't mind

Reply:
@MikeShi42 I went ahead and opened an issue based on your comment to make things better down the line. Hope that's cool with you. Ref: #37

Reply:
Thank you for #37! And it looks like someone already jumped on #36 with a PR (#38)